### DISCUSSION TITLE
Need help for setting up a wifi hotspot on a RPI groundstation

### DISCUSSION THREAD

**Discussion by Yes21** [svpcom/wfb-ng#106] [Questions]
I'm using a raspberry Pi as a ground station, without having QGroundControl running on it.
So I'm need to create a wifi hotspot to be able to connect QGC (running on an android tablet).

That's very easy to do it with EZ-Wifibroadcast 1.6 or Open.HD, because all is preconfigured.
But I don't know how to do it with  svpcom-wifibroadcast .

Setting up a wifi access point is one thing, but the video and telemetry streams should then be redirect to it.

**Any help is welcome.**

ps : It would be useful to all  RPI users ...

**Comment by massimilianonball** [comment]
If you want to see your telemetry data on the video feed on the ground station, you can install OSD on the pi used as groundstation.
https://github.com/svpcom/wifibroadcast_osd
